trezorctl

```diff
@@ -447,10 +447,13 @@ def get_public_node(connect, coin, address, curve, show_display):
 @click.pass_obj
 def sign_tx(connect, coin):
     client = connect()
-    try:
+    if coin in coins_txapi:
         txapi = coins_txapi[coin]
-    except:
-        raise ValueError('Coin "%s" is not supported' % coin)
+    else:
+        print('Coin "%s" is not recognized.' % coin)
```

Comment:
Use `file=sys.stderr`

Reply:
If we want to maintain python 2.x compatibility, I'd have to also add `from __future__ import print_function` at the beginning of the file as [demonstrated here](https://stackoverflow.com/a/14981125/1117929).
I found that `sys.stderr.write()` actually works perfectly on 2.x and 3.x -- seems a bit cleaner as there is no changes to imports / print function, but I'm a python noob, so please advise  :-)

Reply:
I'm fine with whichever is consistent with the rest of the code

Reply:
Okay, went for the `sys.stderr.write()` option then  :-)